### DISCUSSION TITLE
MySql

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#7003] [Help] [answered]
Using embed component :
If I work with a MySql type database, would I have more freedom or is it a limitation via javascript with the platform

**Comment by melohagan** [comment, marked as answer]
Hi @rafikApp 

If you're asking if you can directly use a JS MySQL client within an embed component, you can't because you can't import JavaScript libraries into an Embed.
There is work however to make custom components available within the next quarter which would allow you to build a custom MySQL component.

There is a MySQL connector as well of course.

**Reply by rafikApp** [reply]
Ok. thank you @melohagan
